### DISCUSSION TITLE
How affect use "import type" expression on bundling?

### DISCUSSION THREAD

**Discussion by mrakobes27** [web-infra-dev/rspack#6310] [General]
Hi! Please tell me, does using the “import type” syntax in “typescript” affect the speed of assembling the bundle? and are these syntactic expressions processed at all (I assume they shouldn’t)?

**Comment by xc2** [comment]
when compared to general `import` statement, i believe they won't slow down bundling speed.

afaik, swc just strips type-only imports.
